odoo/addons/base/models/ir_ui_view.py

```diff
@@ -277,6 +277,8 @@ class View(models.Model):
 * if True, the view always extends its parent
 * if False, the view currently does not extend its parent but can be enabled
          """)
+    model_id = fields.Many2one("ir.model", string="Model of the view", compute_sudo=True,
```

Comment:
`compute_sudo=True` is not mandatory anymore because you are using `env['ir.model']._get`.

Reply:
You are right, the idea of `compute_sudo=True` was based on the original algorithm.
Maybe it is simpler and safer to let this one out.